test/engine/test_execute.py

```diff
@@ -272,6 +272,17 @@ def test_raw_named(self, connection):
             (4, "sally"),
         ]
 
+    @testing.requires.named_paramstyle
+    def test_raw_insert_with_empty_list(self, connection):
+        conn = connection
+        assert_raises_message(
+            tsa.exc.ProgrammingError,
+            "Incorrect number of bindings supplied. The current statement uses 2, and there are 0 supplied",
```

Comment:
**Federico Caselli** (CaselIT) wrote:

this statement is too fragile to be like this, since it comes from the db.

let's hardcode to sqlite

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/5101